qemu/tests/cfg/boot_with_max_mem.cfg

```diff
@@ -0,0 +1,7 @@
+- boot_with_max_mem:
+    virt_test_type = qemu
+    only Linux
+    type = boot_with_max_mem
+    start_vm = no
+    boot_mem = 8T
```

Comment:
`mem_fixed = 8T`

Reply:
@yanan-fu , I have updated it, please check it, thanks